ecs_fargate/manifest.json

```diff
@@ -26,7 +26,7 @@
   "assets": {
     "monitors": {},
     "dashboards": {
-      "ECS-Fargate": "assets/dashboards/ecs_fargate_overview.json"
+      "Amazon Fargate": "assets/dashboards/amazon_fargate_overview.json"
```

Comment:
What's the impact of changing the key here?

Reply:
This seems to serve as the short name of the dashboard